pyat/at/physics/orbit.py

```diff
@@ -308,12 +311,16 @@ def find_orbit6(ring, refpts=None, guess=None, **kwargs):
     harm_number = cavities[0].HarmNumber
 
     if guess is None:
-        rfv = ring.periodicity * sum(elem.Voltage for elem in cavities)
-        u0 = get_energy_loss(ring, method=method)
-        if u0 > rfv:
-            raise AtError('Missing RF voltage: unstable ring')
         ref_in = numpy.zeros((6,), order='F')
-        ref_in[5] = -constants.c / (2 * pi * f_rf) * asin(u0 / rfv)
+        try:
+            rfv = ring.get_rf_voltage(cavpts=cavpts)
+        except AtError:
+            pass
```

Comment:
Maybe add a comment here? Otherwise this looks like a not implemented branch.

Reply:
Done.